src/components/ebay-lightbox-dialog/index.marko

```diff
@@ -0,0 +1,21 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+$ var isMini = input.mode === 'mini';
+
+<ebay-dialog-base
+    ...input
+    open=input.open
+    classPrefix="lightbox-dialog"
+    on-modal-open("emit", "lightbox-dialog-open")
```

Comment:
I am concerned that we are using `open` and `show` interchangeably.

Reply:
We should be using `open` now. If you see any more `show` it should be migrated and moved